package bai.zsj.com;

import org.testng.annotations.Test;

public class Test2 {
	@Test
    public void f() throws Exception {
	  System.out.println("成功222");
//	  throw new Exception();
    }
}
